plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -97,6 +96,7 @@ private void addAppEngineLibrariesToBuildPath(IProject newProject, List<Library>
       subMonitor.worked(1);
     }
     javaProject.setRawClasspath(newRawClasspath, monitor);
+    subMonitor.done();
```

Comment:
You don't need to do this with SubMonitor, and I've been encouraged not to include it any more.